multi-buffer-search/src/services/resultFormatter.ts

```
@@ -0,0 +1,163 @@
+        ? ResultFormatter.MATCH_LINE_PREFIX
+        : ResultFormatter.CONTEXT_LINE_PREFIX
+      lines.push(`${formattedLineNum}${prefix}${formattedText}`)
+    }
+
+    return { lines, matchLineIndices }
+  }
+
+  private formatLineNumber(lineNum: number): string {
+    return lineNum.toString().padStart(ResultFormatter.LINE_NUMBER_WIDTH, ' ') +
+      ' '
+  }
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)